www/src/components/__tests__/stub-list.js

```diff
@@ -22,66 +22,18 @@ jest.mock(`../../../utils/sidebar/item-list`, () => {
         },
       ],
     },
-    getItemList: jest.fn(),
   }
 })
-jest.mock(`react-modal`, () => {
-  const modal = jest.fn().mockImplementation(({ children }) => children)
-  modal.setAppElement = jest.fn()
-  return modal
-})
 import React from "react"
-import { useStaticQuery } from "gatsby"
 import { render } from "@testing-library/react"
-import { ThemeProvider } from "theme-ui"
 
-import theme from "../../../../src/gatsby-plugin-theme-ui"
 import StubList from "../stub-list"
 
-let location
-beforeEach(() => {
-  useStaticQuery.mockReturnValueOnce({
-    site: {
-      siteMetadata: {
-        title: `GatsbyJS`,
-      },
-    },
-  })
-  location = {
-    pathname: `/contributing/stub-list`,
-  }
-})
-
 describe(`StubList`, () => {
-  it(`shows a title`, () => {
```

Comment:
Get rid of these tests because they tested the part of the stub list page that's now just markdown